asciidoc/graph-algorithms.adoc

```diff
@@ -8,80 +8,10 @@
 :toc-title: Contents
 :toclevels: 1
 :docs: https://neo4j.com/docs/graph-algorithms/current/algorithms
-:img: https://s3.amazonaws.com/dev.assets.neo4j.com/wp-content/uploads
-
-include::header.adoc[]
-
-++++
```

Comment:
did you check the implications of removing this styling?

Reply:
We only needed it for this page as it has boxes. The other pages don't have these boxes.

Reply:
I moved the styles to the template `embedded`.
I think it's fine to declare all the styles (if they are not conflicting). We usually don't use 100% of the styles declared in a CSS anyway.